docs/user-guide.rst

```diff
@@ -20,23 +20,26 @@ First things first, import the urllib3 module:
 
 .. code-block:: python
 
-    >>> import urllib3
+    import urllib3
 
 You'll need a :class:`~poolmanager.PoolManager` instance to make requests.
 This object handles all of the details of connection pooling and thread safety
 so that you don't have to:
 
 .. code-block:: python
 
-    >>> http = urllib3.PoolManager()
+    http = urllib3.PoolManager()
 
 To make a request use :meth:`~poolmanager.PoolManager.request`:
 
 .. code-block:: python
 
-    >>> r = http.request('GET', 'http://httpbin.org/robots.txt')
-    >>> r.data
-    b'User-agent: *\nDisallow: /deny\n'
+    import urllib3
+    # Send the GET request.
+    r = urllib3.request('GET', 'https://httpbin.org/robots.txt')
+    # Print the returned data.
+    print(r.data)
+    #b'User-agent: *\nDisallow: /deny\n'
```

Comment:
Should be a space after every `#` on a comment line and some whitespace between every section. Also quotes should be Black formatted so double quotes. This example should look like this:

```python
    import urllib3

    # Creating a PoolManager instance for sending requests.
    http = urllib3.PoolManager()

    # Send the GET request and getting back a HTTPResponse object.
    resp = http.request("GET", "https://httpbin.org/robots.txt")

    # Print the returned data.
    print(resp.data)
    # b"User-agent: *\nDisallow: /deny\n"
```

Reply:
Will do.